realm-jni/src/util.hpp

```diff
@@ -58,7 +58,7 @@ JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *reserved);
 
 #define CATCH_FILE(fileName) \
     catch (InvalidDatabase&) { \
-        ThrowException(env, IllegalArgument, "Invalid Group file format."); \
+        ThrowException(env, IllegalArgument, "Invalid format of Realm file."); \
```

Comment:
Does "format" makes any sense to users, it is out of their hands anyway. Wouldn't "Invalid Realm file" or "File is not a valid Realm file" be better?